### ISSUE TITLE
FR-06032022-1 Enhancement to "schedule expire.in.days" filter

### ISSUE THREAD

**Issue by nexusops** [PaloAltoNetworks/pan-os-php#509]
### Is your feature request related to a problem?

Currently the tool allows to search for rules with schedules which expires in X amount of days ("less than" operand). Which means it is not possible to search for rules with schedules which expire after certain amount of days ("greater than" operand).

### Describe the solution you'd like
Is it possible to change
`expire.in.days X`
into
`expire.in.days = > < X`

basically allow the user to choose the operand and then define the number of days

### Describe alternatives you've considered

currently we have to export the resolved schedule values into excel, change the values into a Date format recognisable by Excel and then we can build a filter in Excel

### Additional context

n/a

**Comment by swaschkut** [comment]
implemented with CR:
#515
